piksi_tools/bootload.py

```diff
@@ -78,9 +87,9 @@ def wait_for_handshake(self, timeout=None):
     return True
 
   def reply_handshake(self):
-    # < v2.0 of the bootloader, reuse single handshake message.
-    if self.version < "v2.0":
-      self.link.send(SBP_MSG_BOOTLOADER_HANDSHAKE_DEVICE, '\x00')
+    # < 0.45 of SBP protocol, reuse single handshake message.
+    if self.sbp_version < 45:
```

Comment:
I was comparing it as a 16-bit value, without pulling out the 8-bit minor and 8-bit major. Could be confusing once we rev? E.g., 1.1 = 257.